java/server/src/org/openqa/selenium/grid/sessionqueue/NewSessionQueue.java

```diff
@@ -66,6 +76,40 @@ public boolean hasRequestTimedOut(HttpRequest request) {
     return duration.compareTo(requestTimeout) > 0;
   }
 
+  protected Capabilities validateCaps(Capabilities caps) {
+    String browser = caps.getBrowserName().isEmpty() ? "ANY" : caps.getBrowserName();
```

Comment:
`ANY` is a hangover from the Json Wire Protocol. Worse, since `ANY` won't match any browser according to the new session logic in the w3c spec, this isn't valid. Prefer to omit the field rather than include it.

Reply:
Also, be aware that although the default implementation of `getBrowserName` returns an empty string if there's no match, there's no requirement for this. Add a null check, just in case, or use `Strings.isEmptyOrNull` from guava

Reply:
Thank you for pointing out the usage of `ANY`. Here `'ANY'` is used to help group the new session requests in the queue that do not have specific browser name, platform or version. It is not used to store the requests in the queue, the incoming requests are stored as it is in the queue. I now understand how the `validateCaps` method name might be confusing. I will fix that.